### PR TITLE
Update JDK8 windows compiles to use Visual Studio 2013

### PR DESCRIPTION
**Pull request by jdekonin**
* [skip ci]
* disable the use of rebase
* eclipse/openj9#10129

Signed-off-by: Joe deKoning <joe_dekoning@ca.ibm.com>

### CODE REVIEW

**Comment by jdekonin** [comment]
Needs to merge with https://github.com/eclipse/openj9/pull/10892

File: closed/OpenJ9.gmk
Comment (keithc-ca) [inline closed/OpenJ9.gmk:384]: Why is this added? I don't see any dependency on that macro in `buildtools.mk`.

File: common/autoconf/generated-configure.sh
Comment (keithc-ca) [inline common/autoconf/generated-configure.sh:1017]: Can you regenerate on a newer system so we don't get these unrelated changes?

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: closed/OpenJ9.gmk
Comment (jdekonin) [reply closed/OpenJ9.gmk:384]: I thought I had traced this back to a previous PR as being required, but retesting now show that its not needed.

File: common/autoconf/generated-configure.sh
Comment (jdekonin) [reply common/autoconf/generated-configure.sh:1017]: Done

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
@pshipton I'll let you merge this and eclipse/openj9#10892 when you get satisfactory answers from Joe.

**Review by pshipton — APPROVED** [review]

**Comment by pshipton** [comment]
Hmm, I'm thinking that once this is merged all builds need to use VS2013 or they will fail. That include builds at Adopt. https://github.com/AdoptOpenJDK/openjdk-build/issues/466#issuecomment-717348184

**Comment by pshipton** [comment]
@jdekonin there is a merge conflict here.

**Comment by jdekonin** [comment]
Rebase performed and conflict resolved @pshipton

**Pull request merged**
